extensions/Worker.Extensions.Rpc/src/GrpcHttpClientBuilderExtensions.cs

```diff
@@ -31,7 +31,18 @@ public static IHttpClientBuilder ConfigureForFunctionsHostGrpc(this IHttpClientB
 
             ValidateGrpcClient(builder);
             builder.Services.AddOptions<GrpcClientFactoryOptions>(builder.Name)
-                .Configure<IConfiguration>((options, config) => options.Address = config.GetFunctionsHostGrpcUri());
+                .Configure<IConfiguration>((options, config) =>
+                {
+                    options.Address = config.GetFunctionsHostGrpcUri();
+                    if (config.GetFunctionsHostMaxMessageLength() is int length)
+                    {
+                        options.ChannelOptionsActions.Add(o =>
+                        {
+                            o.MaxReceiveMessageSize = length;
+                            o.MaxSendMessageSize = length;
```

Comment:
Today, this defaults to `null`, and there will be no message limits enforced, correct?

The host will pass `grpcMaxMessageLength` by default, so this change will start setting a limit here, defaulting to `int.MaxValue`, so  ~2GB, which is large, but change in behavior to be aware of here.

Reply:
The default is 4MB, not null. This will increase the limit.

Reply:
@jviau can you double check this? A resolved instance of `GrpcChannelOptions` will have the `MaxSendMessageSize` set to `null` with the OOB setup.